src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -155,7 +156,21 @@ private BackgroundPreviewActiveState GetBackgroundPreviewData(string name)
         /// <summary>
         /// Should the background grid be shown?
         /// </summary>
-        public bool IsBackgroundGridVisible { get; set; }
+        public bool IsBackgroundGridVisible
+        {
+            get
+            {
+                return isBackgroundGridVisible;
+            }
+            set
+            {
+                if (value == isBackgroundGridVisible) return;
+                isBackgroundGridVisible = value;
+
+                RaisePropertyChanged("IsBackgroundGridVisible");
```

Comment:
can we use `nameof`?

Reply:
Good catch. Now that we're on C# 6.0 we should add a task to switch all all our  `RaisePropertyChanged("")` calls to use `nameof`.